### ISSUE TITLE
`<input type="search">` provides a clear button "X" that can't be customized

### ISSUE THREAD

**Issue by argyleink** [openui/open-ui#873]
see https://codepen.io/argyleink/pen/VwqgRjX (type in something and see the X show up)

there's also accent-color to consider, see this demo https://codepen.io/argyleink/pen/qBRwjZm

I'd like to see standardization around this clear button, allowing basic styling and positioning (as well as hit area improving, cuz it can suck to try and tap with an adult finger)

**Comment by lukewarlow** [comment]
I agree this would be a big win. I'm working on pushing research for search soon based on my research into browsers pseudos. There's also the magnifying glass that some browsers add to the search input too.

This along with `<input type="number">` controls would be a big win.

**Comment by lukewarlow** [comment]
Now I want to make sure we don't fall in the trap of letting invokers solve problems we should be fixing in CSS, that being said I wonder if a clear action would fit into the invokers paradigm so you can just make your own styled button? And hide the existing one with CSS?

**Comment by scottaohara** [comment]
definitely would rather authors could do more with CSS for this 'button', rather than having to create a whole separate button and understand the nuance of making a button that is not generally expected to be overtly exposed to users (AT or keyboard in general) now have reasonable a11y-ux.

**Comment by lukewarlow** [comment]
Do you happen to have a similar post about steppers for number input. I'd love to understand the accessibility implications. So far all I can find is that you should probably put tabindex=-1.

**Comment by lukewarlow** [comment]
Per https://drafts.csswg.org/css-forms-1/#selectordef-clear-icon and the recent decision to add a reset/clear command to the future invokers list I think this is addressed?
